MyBus/MyBus/SuggestionSearchViewController.swift

```diff
@@ -10,74 +10,139 @@ import UIKit
 import Mapbox
 import RealmSwift
 
-public enum SearchFilterType {
-    case Search,Favorite,Tourist
+protocol SuggestionProtocol {
```

Comment:
Maybe in the future we could add a **"func getPoint()->RoutePoint?"** method in the protocol to reduce the creation code in didSelectRowAtIndexPathMethod  of the tableView